tests/opentelemetry-test-utils/src/opentelemetry/test/test_base.py

```diff
@@ -36,6 +42,11 @@ def setUpClass(cls):
         reset_trace_globals()
         trace_api.set_tracer_provider(cls.tracer_provider)
 
+        result = cls.create_meter_provider()
```

Comment:
I see we do the same thing for tracer provider, but what is the reason to have this in setUpClass instead of setUp? Seems like the test cases could introduce side effects to each other.

Reply:
I am not totally sure the reason behind doing it in setUpClass. I can check the commit history to see if I can find something.